src/app/features/add-network/add-network.tsx

```diff
@@ -44,13 +45,15 @@ export function AddNetwork() {
                   >
                     <styled.span textStyle="body.02">
                       Use this form to add a new instance of the{' '}
-                      <Link
-                        href="https://github.com/blockstack/stacks-blockchain-api"
-                        target="_blank"
-                        rel="noreferrer"
-                      >
-                        Stacks Blockchain API
-                      </Link>{' '}
+                      <Box display="inline-block">
```

Comment:
That is a weird issue. I spent a few minutes trying different ways but didn't come up with anything quickly. 

I think it's related to us using `<Link` inside of the `<span`. We could probably refactor this into it's own component and spot the issue faster as it's a bit weird anyway now.

Removing the `Box` and the first `<Link` seems to take up loads of space:
![Screenshot 2024-09-25 at 09 48 10](https://github.com/user-attachments/assets/60610e64-d3c6-4c25-bef9-0a0cd72a1f53)

It would be good to add a better solution but what you have done is working already so it could be fine